lib/config/validation.ts

```diff
@@ -269,6 +270,18 @@ export async function validateConfig(
             if (key === 'extends') {
               for (const subval of val) {
                 if (is.string(subval)) {
+                  try {
+                    await getPreset(subval, config);
+                  } catch (err) {
+                    logger.warn(
+                      { err },
+                      `Preset Error while validating ${subval}`
```

Comment:
Avoid using templates within a warning log message. Otherwise it's not easy to group/aggregate these warnings together on the backend. Put it in metadata instead.

```suggestion
                      `Could not resolve preset during config validation`
```